frontend/packages/dev-console/integration-tests/support/pages/functions/installOperatorOnClusterUsingCLI.ts

```diff
@@ -123,38 +129,47 @@ export const installOperatorUsingCLI = (operator: operators) => {
 export const checkSubscriptionStatus = (operator: operators) => {
   let namespace;
   let subscriptionName;
+  let operatorPackageName;
   const resourceName = 'subscriptions.operators.coreos.com';
   const condition = 'CatalogSourcesUnhealthy=false';
 
   switch (operator) {
     case operators.PipelinesOperator:
+      operatorPackageName = operatorPackage.PipelinesOperator;
       namespace = operatorNamespaces.PipelinesOperator;
       subscriptionName = operatorSubscriptions.PipelinesOperator;
       break;
     case operators.ServerlessOperator:
+      operatorPackageName = operatorPackage.ServerlessOperator;
       namespace = operatorNamespaces.ServerlessOperator;
       subscriptionName = operatorSubscriptions.ServerlessOperator;
       break;
     case operators.ShipwrightOperator:
+      operatorPackageName = operatorPackage.ShipwrightOperator;
       namespace = operatorNamespaces.ShipwrightOperator;
       subscriptionName = operatorSubscriptions.ShipwrightOperator;
       break;
     case operators.BuildsForOpenshiftOperator:
+      operatorPackageName = operatorPackage.BuildsForOpenshiftOperator;
       namespace = operatorNamespaces.BuildsForOpenshiftOperator;
       subscriptionName = operatorSubscriptions.BuildsForOpenshiftOperator;
       break;
     case operators.WebTerminalOperator:
+      operatorPackageName = operatorPackage.WebTerminalOperator;
       namespace = operatorNamespaces.WebTerminalOperator;
       subscriptionName = operatorSubscriptions.WebTerminalOperator;
       break;
     case operators.RedHatIntegrationCamelK:
+      operatorPackageName = operatorPackage.RedHatIntegrationCamelK;
       namespace = operatorNamespaces.RedHatIntegrationCamelK;
       subscriptionName = operatorSubscriptions.RedHatIntegrationCamelK;
       break;
     default:
       throw new Error('Invalid Operator');
   }
 
+  checkOperatorvailabilityStatus(operatorPackageName);
+
```

Comment:
If an operator is already installed it will recheck its availability making it redundant. Keep this as part else statement in the cy.exec() where it is  checking for operator subscription